src/Components/MainMenu/MainMenu.js

```diff
@@ -1,17 +1,32 @@
 import React from 'react'
 import {Link} from 'react-router-dom'
+import Media from 'react-media'
 import './MainMenu.css'
+import HamburgerButton from '../../Views/HamburgerButton/HamburgerButton';
 
 export default class MainMenu extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = { visible: false };
+        this.toggleNavigation = this.toggleNavigation.bind(this);
+    }
+
+    toggleNavigation = () => {
+        this.setState((state) => ({
+            visible: !state.visible
+        }));
+    }
+
     render () {
         return (
-            <div className= "header">
-                 <div className="menu-list">
+            <header className={this.state.visible? "nav-on": "nav-off"}>  
+            <Media query="(max-width: 780px)" render={() => <HamburgerButton action={this.toggleNavigation}/>} />
```

Comment:
Please set the outline:none for the Hamburger Button

Reply:
Cool. Will add a commit to this effect.